app/models/rapidfire/question.rb

```diff
@@ -2,6 +2,9 @@ module Rapidfire
   class Question < ActiveRecord::Base
     belongs_to :survey, :inverse_of => :questions
     has_many   :answers
+    has_many   :conditionals, dependent: :destroy
+    has_many   :question_groups, through: :question_groups_questions, inverse_of: :questions
+    has_many   :question_groups_questions, dependent: :nullify, inverse_of: :question
```

Comment:
`dependent: :destroy` you don't what to have a question_groups_questions with only group id